products/apache-http-server.md

```diff
@@ -29,25 +29,28 @@ releases:
     eol: false
     latest: "2.4.63"
     latestReleaseDate: 2025-01-23
-    link: https://downloads.apache.org/httpd/Announcement2.4.html
+    link: https://downloads.apache.org/httpd/CHANGES_2.4.63
```

Comment:
You could have used `https://downloads.apache.org/httpd/CHANGES___LATEST__`.

More generally @usta I noticed that [most of your latest PRs are merged without any review](https://github.com/endoflife-date/endoflife.date/pulls?q=is%3Apr+author%3Austa+). Could you follow the usual review process ? Even for simple changes it can avoid small typo / mistakes.